lte/gateway/fabfile.py

```diff
@@ -12,19 +12,14 @@
 """
 
 import sys
-from datetime import datetime
 from time import sleep
 
-from fabric.api import cd, env, execute, lcd, local, run, settings, sudo
-from fabric.contrib.files import exists
-from fabric.operations import get
-from fabric.utils import error, fastprint, puts
-
 sys.path.append('../../orc8r')
 import tools.fab.pkg as pkg
+# import fab tasks from dev_tools, so they can be called via fab in the command line
```

Comment:
Ah I see - that's the reason why `-f dev_tools.py` can be removed in various places.

Reply:
the `-f {fabfile}` flag just doesn't exist anymore (`-f` is now used to specify a configuration file)